app/assets/javascripts/components/admin/unmatched_page.es6.jsx

```diff
@@ -27,11 +28,12 @@ class UnmatchedPage extends React.Component {
     );
   }
 
-  studentOnClick(studentId) {
+  studentOnClick(studentId, instrument) {
     var studentRoute = ApiConstants.students.show(studentId)
     var studentResolve = ((response) => {
       this.setState({fullStudent: true, student: response});
-      var teacherRoute = ApiConstants.teachers.possibleTeachers(studentId);
+      this.setState({instrument: instrument.name});
```

Comment:
combine this setstate with the one on top so 

`this.setState({ 
       fullStudent: true, 
        student: response,
        instrument: instrument.name,
})`